js/ui/data_grid/ui.data_grid.focus.js

```diff
@@ -80,6 +80,10 @@ gridCore.registerModule("focus", extend(true, {}, focusModule, {
                         groupPath,
                         group = dataSource.group();
 
+                    if(Array.isArray(key)) {
```

Comment:
For better code readability use a variable "isGroupKey" initialized by this condition